textual/src/toga_textual/dialogs.py

```diff
@@ -1,12 +1,11 @@
 from pathlib import Path
 
-from toga_textual.window import TitleBar
-
 import toga
 from textual.app import ComposeResult
 from textual.containers import Horizontal, Vertical, VerticalScroll
 from textual.screen import ModalScreen
 from textual.widgets import Button, DirectoryTree, Input, Label, Static
+from toga_textual.window import TitleBar
```

Comment:
I think we've missed `toga_textual` as known-first-party. Not sure why `toga` isn't being picked up as first-party.